Sources/NIO/SelectableEventLoop.swift

```diff
@@ -53,7 +53,7 @@ internal final class SelectableEventLoop: EventLoop {
         case noLongerRunning
     }
 
-    private let selector: NIO.Selector<NIORegistration>
+    internal let _selector: NIO.Selector<NIORegistration>
```

Comment:
consider adding a comment that this is not private just because of testing.

Reply:
thanks, done